modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,111 @@
+
+.Procedure
+
+. Edit the `install-config.yaml` configuration file so that it contains the `credentialsMode` parameter set to `Manual`.
++
+.Example `install-config.yaml` configuration file
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Manual`.
+
+. To generate the manifests, run the following command from the directory that contains the installation program:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
+----
+
+. From the directory that contains the installation program, obtain the {product-title} release image that your `openshift-install` binary is built to use:
++
+[source,terminal]
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
+. Extract the `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal]
+----
+$ oc adm release extract --cloud=ibmcloud --credentials-requests $RELEASE_IMAGE \
+    --to=<path_to_credential_requests_directory> <1>
+----
+<1> The directory where the credential requests are stored.
++
+This command creates a YAML file for each `CredentialsRequest` object.
++
+.Sample `CredentialsRequest` object
++
+[source,yaml]
+----
+  apiVersion: cloudcredential.openshift.io/v1
+  kind: CredentialsRequest
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: openshift-image-registry-ibmcos
+    namespace: openshift-cloud-credential-operator
+  spec:
+    secretRef:
+      name: installer-cloud-credentials
+      namespace: openshift-image-registry
+    providerSpec:
+      apiVersion: cloudcredential.openshift.io/v1
+      kind: IBMCloudProviderSpec
+      policies:
+      - attributes:
+        - name: serviceName
+          value: cloud-object-storage
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+        - crn:v1:bluemix:public:iam::::role:Operator
+        - crn:v1:bluemix:public:iam::::role:Editor
+        - crn:v1:bluemix:public:iam::::serviceRole:Reader
+        - crn:v1:bluemix:public:iam::::serviceRole:Writer
+      - attributes:
+        - name: resourceType
+          value: resource-group
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+----
+
+. Create the service ID for each credential request, assign the policies defined, create an API key in IBM Cloud, and generate the secret:
++
+[source,terminal]
+----
+$ ccoctl ibmcloud create-service-id \
+    --credentials-requests-dir <path_to_store_credential_request_templates> \ <1>
+    --name <name> \ <2>
+    --resource-group-name <resource_group_name> \ <3>
+    --output-dir <installation_directory>
+----
+<1> The directory to store credential request templates.
+<2> The name of the {product-title} cluster.
+<3> Optional: The resource group name of the cluster.
```

Comment:
Details on how to find the resource group name could be helpful here.  If an incorrect name is provided, the installation will stall and fail in bootstrap(this bit me).  Something like:
~~~
cat <installation_directory>/manifests/cluster-infrastructure-02-config.yml | grep resourceGroupName: | cut -d':' -f2
~~~

Reply:
Hi @rvanderp3, that parameter is completely optional and mostly used when you are using a custom `resourceGroupName`, but even with that, it's still optional, it just narrows down the scope of the credential within the account if needed.
@codyhoag, maybe we can remove it from the example and put it in a separate section/note to simplify the process, what do you think?

Reply:
Right, my concern is that if it's used and it's wrong, it is nearly impossible to determine that that is the problem without a firm understanding of what is happening in the bootstrap process.  I'd be fine with moving it to a different section where the caveats and examples can be provided.

Reply:
Thank you. I have added more info regarding this suggestion (and the others recommended previously). Let me know if you have any additional feedback.